backend/api/v1/project_service.go

```diff
@@ -425,6 +428,11 @@ func (s *ProjectService) SetIamPolicy(ctx context.Context, request *v1pb.SetIamP
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
+
+	if bd, ok := ctx.Value(common.BindingDeltaKey).(*[]*v1pb.BindingDelta); ok {
```

Comment:
Instead of calling BindingDeltaKey, let's call it ServiceDataKey and pass in the service data Any. This interface will be generic enough for future service data integrations.